examples/src/main/scala/caliban/ExampleApp.scala

```diff
@@ -48,9 +57,10 @@ object ExampleApp extends CatsApp with GenericSchema[Console with Clock] {
       _ <- BlazeServerBuilder[ExampleTask]
             .bindHttp(8088, "localhost")
             .withHttpApp(
-              Router(
+              Router[ExampleTask](
                 "/api/graphql" -> CORS(Http4sAdapter.makeRestService(interpreter)),
-                "/ws/graphql"  -> CORS(Http4sAdapter.makeWebSocketService(interpreter))
+                "/ws/graphql"  -> CORS(Http4sAdapter.makeWebSocketService(interpreter)),
+                "/graphiql" -> Kleisli.liftF(OptionT.liftF(staticResource("graphiql.html")))
```

Comment:
This code can be a bit simpler no? In https://http4s.org/v0.21/static/ it seems we can simply do something like `StaticFile.fromResource("/" + file, blocker, Some(request)).getOrElseF(NotFound())`

Reply:
Fixed